doc/nrf/protocols/thread/certification.rst

```diff
@@ -177,3 +180,48 @@ This is valid only for Nordic Semiconductor development kits with a J-Link virtu
 
 To add an nRF52840 DK, drag the nRF52840 DK and drop it on the test bed configuration page.
 After that, the device is configured and the :ref:`proper baud rate (115200) <test_and_optimize>` and COM port are set.
+
+.. _ug_thread_build_report:
+
+OpenThread build report
+***********************
+
+The OpenThread build report contains information about the current |NCS| and OpenThread revisions, Thread feature set, Thread library path and changes in the :ref:`nrfxlib:nrfxlib` repository in comparison with the latest |NCS| release.
+The report is generated to the output console log, and stored as an additional build artefact in the application build directory.
+
+To enable building the Thread report set the :kconfig:option:`CONFIG_OPENTHREAD_REPORT` kconfig option to ``y``.
```

Comment:
Shouldn't we just enable it by default?

Reply:
It may be default if for example L2_OPENTHREAD config is enabled. @LuDuda what do you think, do you want to have it enabled by default?